content/zh-cn/blog/_posts/2022-08-29-csi-inline-volumes-ga.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: blog
+title: "Kubernetes 1.25: CSI 内联存储卷迎来 GA 正式发布版本"
+date: 2022-08-29
+slug: csi-inline-volumes-ga
+---
+<!--
+layout: blog
+title: "Kubernetes 1.25: CSI Inline Volumes have graduated to GA"
+date: 2022-08-29
+slug: csi-inline-volumes-ga
+-->
+
+<!--
+**Author:** Jonathan Dobson (Red Hat)
+-->
+**作者：** Jonathan Dobson (Red Hat)
+
+<!--
+CSI Inline Volumes were introduced as an alpha feature in Kubernetes 1.15 and have been beta since 1.16. We are happy to announce that this feature has graduated to General Availability (GA) status in Kubernetes 1.25.
+
+CSI Inline Volumes are similar to other ephemeral volume types, such as `configMap`, `downwardAPI` and `secret`. The important difference is that the storage is provided by a CSI driver, which allows the use of ephemeral storage provided by third-party vendors. The volume is defined as part of the pod spec and follows the lifecycle of the pod, meaning the volume is created once the pod is scheduled and destroyed when the pod is destroyed.
+-->
+CSI 内联存储卷是在 Kubernetes 1.15 中作为 alpha 功能推出的，并从 1.16 开始成为 beta 版本。
+我们很高兴地宣布，这项功能在 Kubernetes 1.25 版本中已经升级为通用（GA）状态。
+
+CSI 内联存储卷与其他临时卷类型相似，如  `configMap`、`downwardAPI` 和  `secret`。
+重要的区别是，存储是由 CSI 驱动提供的，它允许使用第三方供应商提供的临时存储。
+卷被定义为 Pod spec 的一部分，并遵循 Pod 的生命周期，这意味着卷随着 Pod 的调度而创建，并随着 Pod 的销毁而销毁。
+
+<!--
+## What's new in 1.25?
+
+There are a couple of new bug fixes related to this feature in 1.25, and the [CSIInlineVolume feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) has been locked to `True` with the graduation to GA. There are no new API changes, so users of this feature during beta should not notice any significant changes aside from these bug fixes.
+
+- [#89290 - CSI inline volumes should support fsGroup](https://github.com/kubernetes/kubernetes/issues/89290)
+- [#79980 - CSI volume reconstruction does not work for ephemeral volumes](https://github.com/kubernetes/kubernetes/issues/79980)
+-->
+
+## 1.25 版本有什么新内容？
+
+在 1.25 版本中有几个新的与该功能相关的错误修复，
+并且 [CSIInlineVolume 特性门控](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
```suggestion
[并且 [CSIInlineVolume 特性门控](https://kubernetes.io/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```

Reply:
Why is there a symbol at the beginning?
Do I still need to keep the "https://kubernetes.io/" before the ”/zh-cn“ ?  I saw a similar suggestion above that was not retained.